sonic-xcvrd/tests/test_y_cable_helper.py

```diff
@@ -0,0 +1,1839 @@
+from xcvrd.xcvrd_utilities.port_mapping import *
```

Comment:
check the imports again, and only import what is required

Reply:
Reason for leaving this in the file is mentioned above.